metadata-io/src/main/java/com/linkedin/metadata/client/EntityClientAspectRetriever.java

```diff
@@ -52,4 +55,25 @@ public Map<Urn, Map<String, Aspect>> getLatestAspectObjects(
       }
     }
   }
+
```

Comment:
Can the duplicated method here be shared between the client & server retrievers?

Reply:
maybe, i'll see